Extensions/mass_plus.js

```diff
@@ -171,7 +171,7 @@ XKit.extensions.mass_plus = new Object({
 		if (last_timestamp.indexOf(" ") !== -1) {
 			last_timestamp = last_timestamp.substring(0, last_timestamp.indexOf(" "));
 		}
-		var blog_shortname = location.pathname.split("/").slice(-1)[0];
+		var blog_shortname = location.pathname.split("/")[3];
```

Comment:
might be clearer as array destructuring indicating what you expect the other components of the url to be? Something like:

`var [_root, _page, _post_type, blog_shortname] = location.pathname.split("/")`

Reply:
had to not actually assign variables because of `no-unused-vars` but they're labelled now!